images/macos/software-report/SoftwareReport.Generator.ps1

```diff
@@ -175,7 +175,11 @@ if ( -not $os.IsHighSierra) {
     $markdown += New-MDHeader "Rust Tools" -Level 3
     $markdown += New-MDList -Style Unordered -Lines @(
         (Get-RustVersion),
-        (Get-RustupVersion)
+        (Get-RustupVersion),
+        (Get-RustfmtVersion),
+        (Get-RustdocVersion),
+        (Get-CargoVersion),
+        (Get-ClippyVersion)

```

Comment:
I think `Rustfmt` and `RustClippy` should be in the packages section. See Ubuntu docs: https://github.com/actions/virtual-environments/blob/main/images/linux/Ubuntu1804-README.md#rust-tools

But, I could be wrong

Reply:
You are right - they are installed with the package managed so they should be considered to be packages, thanks